spec/models/sms/sms_decoder_spec.rb

```diff
@@ -351,7 +351,7 @@
       assert_decoding(body: "#{@form.code} 1.15 2.11,12,13,14,15,20,1a", answers: [15, %w(1 2 3 4 5 36 10)])
     end
 
-    it "decimal question should work", :investigate do
+    it "decimal question should work" do
```

Comment:
The :investigate tag was for flapping specs that usually pass but not always. Do these always pass now?